src/set_theory/game.lean

```diff
@@ -12,6 +13,13 @@ In this file we define the quotient of pre-games by the equivalence relation `p
 p`, and construct an instance `add_comm_group game`, as well as an instance `partial_order game`
 (although note carefully the warning that the `<` field in this instance is not the usual relation
 on combinatorial games).
+
+## Multiplication on pre-games
+
+We define the operations of multiplication and inverse on pre-games, and prove a few basic theorems
+about them. Multiplication is not well-behaved under equivalence of pre-games i.e. `x.equiv y` does
+not imply `(x*z).equiv (y*z)`. Hence, multiplication is not a well-defined operation on games.
+Nevertheless, the abelian group structure on games allows us to simplify many proofs for pre-games.
```

Comment:
I don't understand what this sentence has to do with multiplication.

Reply:
My plan is to provide several `simp` lemmas of the form 

```lean 
theorem quot_mul_zero (x : pgame) : ⟦x * 0⟧ := ⟦x⟧ := sorry
```

and wanted to indicate this somewhere in the docstring. 
We can remove this sentence for now and add a docstring later when I add these `simp` lemmas.